protocol-configuration/mainnetAddresses.ts

```diff
@@ -2508,6 +2547,11 @@ export const MainnetContractsConfig = {
     address: '0x14d2F529576d3a5451BfbA370408a899Bb5261a9',
     category: AddressCategory.Deprecated
   },
+  ohmEscrowUnwind: {
+    artifactName: 'OtcEscrow',
+    address: '0x01e292c0fBe4C35aE3af981a37Af30ca86d5a712',
+    category: AddressCategory.Deprecated
```

Comment:
Why is this deprecated?

Reply:
It would be used as soon as its executed so may as well just be deprecated is my guess. External or tbd would also work